x-pack/plugins/cases/server/client/cases/update.ts

```diff
@@ -469,38 +463,41 @@ export const update = async (
       );
     }
 
-    const updateCases: ESCasePatchRequest[] = query.cases.map((updateCase) => {
-      const currentCase = myCases.saved_objects.find((c) => c.id === updateCase.id);
-      const { connector, ...thisCase } = updateCase;
-      return currentCase != null
-        ? getCaseToUpdate(currentCase.attributes, {
-            ...thisCase,
-            ...(connector != null
-              ? { connector: transformCaseConnectorToEsConnector(connector) }
-              : {}),
-          })
-        : { id: thisCase.id, version: thisCase.version };
-    });
+    const updateCases: UpdateRequestWithOriginalCase[] = query.cases.reduce(
```

Comment:
This block now zips togethers the original case if it existed and the updated request. This is particularly useful for when we call the `patch` request so that we can populate the `originalCase` field. If the original case cannot be found within the map we skip the request. This shouldn't even happen though because the code above should handle finding requests that don't exist.